### PR TITLE
ISO week and ISO year functions

### PR DESCRIPTION
**Pull request by LeshaKoss**

### CODE REVIEW

File: src/get_iso_year.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+var startOfWeek = require('./start_of_week');
+
+/**
+ * Returns ISO week-numbering year of given date,
+ * which always starts 3 days before year's first Thursday.
+ * @param {date|string} dirtyDate
+ * @returns {number} (ISO year)
```
Comment (kossnocorp) [inline src/get_iso_year.js]: Please add an example

File: src/get_iso_week.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+var startOfISOYear = require('./start_of_iso_year');
+
+/**
+ * Returns ISO week of given date.
+ * @param {date|string} dirtyDate
+ * @returns {number} (ISO week)
```
Comment (kossnocorp) [inline src/get_iso_week.js]: Example needed

File: src/set_iso_week.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * Sets ISO week to given date, saving weekday number.
+ * @param {date|string} dirtyDate
+ * @param {number} isoWeek
+ * @returns {date} (new date)
+ */
+var setISOWeek = function(dirtyDate, isoWeek) {
+  var date = new Date(dirtyDate);
+  var diff = getISOWeek(date) - isoWeek;
+  date.setTime(date.getTime() - diff * 604800000);
```
Comment (kossnocorp) [inline src/set_iso_week.js]: Please avoid [magic numbers](http://en.wikipedia.org/wiki/Magic_number_(programming)), use constant instead eg `var WHAT_DOESN_THIS_NUMBER_MEAN = 604800000;`

**Comment by LeshaKoss** [comment]
Check out an update

**Comment by kossnocorp** [comment]
Awesome, thanks.

**Pull request merged**